blocks/header/header.js

```diff
@@ -155,7 +155,7 @@ function createSearchBox() {
     const searchInputContainer = div({ class: 'search-input-container' });
     const searchInputBox = document.createElement('input');
     const searchIcon = img({ class: 'search-icon' });
-    searchIcon.src = '../icons/search-white.svg';
+    searchIcon.src = '/icons/search-white.svg';
```

Comment:
Can you try `${window.hlx.codeBasePath}/icons/search-white.svg` and see if that works?

Reply:
Its Working